docs/marionette.functions.md

```diff
@@ -14,17 +14,12 @@ a way to get the same behaviors and conventions from your own code.
 * [Marionette.isNodeAttached](#marionetteisnodeattached)
 * [Marionette.mergeOptions](#marionettemergeoptions)
 * [Marionette.getOption](#marionettegetoption)
-* [Marionette.proxyGetOption](#marionetteproxygetoption)
 * [Marionette.triggerMethod](#marionettetriggermethod)
 * [Marionette.bindEvents](#marionettebindevents)
-* [Marionette.triggerMethodOn](#marionettetriggermethodon)
+* [Marionette.bindRequests](#marionettebindrequests)
```

Comment:
I think the whole thing looks fine to me.. personal preference.. I think it'd be better for `bindEvents` and `unbindEvents` to be grouped together.. and for the requests to be grouped together as it is by subject and the source is also grouped in the same file... so if you're looking up something on requests.. both interfaces are documented together..  I can understand the reasoning for grouping it the way you did though..but I think it is less likely that someone needs to look up how to unbind "things" and will want to see that grouped together.